modules/installation-mirror-repository.adoc

```diff
@@ -99,8 +99,9 @@ $ ARCHITECTURE=<server_architecture>
 +
 [source,terminal]
 ----
-$ REMOVABLE_MEDIA_PATH=<path>
+$ REMOVABLE_MEDIA_PATH=<path> <1>
 ----
+<1> Specify the full path, including the initial "/" (slash) character.
```

Comment:
Should `"/"` be in backticks instead of quotes since it's part of a file path, or not in this context?

Reply:
I lifted it from a neighboring module, but it's not the right way to describe a character.

> Do not refer to an individual punctuation mark or special character by its symbol alone. Use the name alone, or use the name followed by the symbol in parentheses.

So it should be:
> the initial forward slash (/)

Reply:
Sorry to open a can of worms :) Seems like a small can at least!